app/webhook_test.go

```diff
@@ -270,19 +264,19 @@ func TestUpdateIncomingWebhook(t *testing.T) {
 			th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnablePostIconOverride = tc.EnablePostIconOverride })
 
 			updatedHook, err := th.App.UpdateIncomingWebhook(hook, &tc.IncomingWebhook)
-			if tc.ExpectedError && err == nil {
-				t.Fatal("should have failed")
-			} else if !tc.ExpectedError && err != nil {
-				t.Fatalf("should not have failed: %v", err.Error())
+			if tc.ExpectedError {
+				assert.NotNil(t, err, "should have failed")
```

Comment:
Same in both branches of this conditional

Reply:
fixed